ThingIFSDK/ThingIFSDKTests/PostNewCommandWithCommandFormTests.swift

```diff
@@ -0,0 +1,364 @@
+//
+//  PostNewCommandWithCommandFormTests.swift
+//  ThingIFSDK
+//
+//  Created on 2016/04/11.
+//  Copyright 2016 Kii. All rights reserved.
+//
+
+import XCTest
+@testable import ThingIFSDK
+
+class PostNewCommandWithCommandFormTests: SmallTestBase {
+    override func setUp() {
+        super.setUp()
+    }
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    static func equalDictionary(
```

Comment:
It is used several times in different test classes. Better to abstract this method as util method.